nixos/modules/rename.nix

```diff
@@ -233,6 +233,7 @@ with lib;
     (mkRemovedOptionModule [ "services" "mysql" "rootPassword" ] "Use socket authentication or set the password outside of the nix store.")
     (mkRemovedOptionModule [ "services" "zabbixServer" "dbPassword" ] "Use services.zabbixServer.database.passwordFile instead.")
     (mkRemovedOptionModule [ "systemd" "generator-packages" ] "Use systemd.packages instead.")
+    (mkRemovedOptionModule [ "services" "httpd" "extraSubservices" ] "Most existing subservices have been ported to the NixOS module system. Please update your configuration accordingly.")
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/61570, I think this would be better in the relevant module directly

Reply:
I didn't know about that PR. Thank you, I'll update accordingly.